input/intro-notes/StructureDefinition-au-coverage-intro.md

```diff
@@ -2,3 +2,11 @@
 
 **Profile specific implementation guidance:**
 - See each Identifier profile page for guidance related to that identifier type.
+- When populating the type using the preferred ValueSet :
+    - *Medicare* coverage **SHOULD** be represented by sending the V3 Value SetActCoverageTypeCode code "PUBLICPOL"
+    - *DVA* coverage **SHOULD** be represented by sending the V3 Value SetActCoverageTypeCode code "VET"
```

Comment:
Should be 

*DVA* coverage **SHOULD** be represented by sending the  v3.ActCode code "VET"

Reply:
I am unable to make changes to the resolution.